api/user.go

```diff
@@ -2592,17 +2592,29 @@ func sanitizeProfile(c *Context, user *model.User) *model.User {
 }
 
 func searchUsers(c *Context, w http.ResponseWriter, r *http.Request) {
-	props := model.MapFromJson(r.Body)
+	props := model.StringInterfaceFromJson(r.Body)
```

Comment:
Can we unmarshal this into a struct instead? (could even be anonymous if you want)